sechub-adapter-pds/src/main/java/com/daimler/sechub/adapter/pds/PDSWebScanConfigImpl.java

```diff
@@ -82,6 +82,22 @@ public final PDSWebScanConfigBuilder setJobParameters(Map<String, String> jobPar
         @Override
         protected void customBuild(PDSWebScanConfigImpl config) {
             jobParameters.put(PDSDefaultParameterKeyConstants.PARAM_KEY_PDS_SCAN_TARGET_TYPE, config.getTargetType());
+            
+            // to get the base URl from the URL
+            // https://example.org/abc -> https://example.org
+            
+            try
+            {            
+            	URL url = new URL(PARAM_KEY_PDS_SCAN_TARGET_URL);
```

Comment:
The logic does not work here.

`PARAM_KEY_PDS_SCAN_TARGET_URL` is just a constant and represents a key which will be later available as an ENV entry inside PDS runtime.

`config.getTargetAsString());` does contain the URL as a simple string.
So if we would implement the behaviour - we no longer need the feature - we would simply do some string operations here.